ui/app/services/flags.ts

```diff
@@ -81,31 +79,4 @@ export default class flagsService extends Service {
   fetchActivatedFlags() {
     return this.getActivatedFlags.perform();
   }
-
-  @lazyCapabilities(apiPath`sys/activation-flags/secrets-sync/activate`)
-  declare secretsSyncActivatePath: CapabilitiesModel;
-
-  get canActivateSecretsSync() {
-    return (
-      this.secretsSyncActivatePath.get('canCreate') !== false ||
-      this.secretsSyncActivatePath.get('canUpdate') !== false
-    );
-  }
-
-  get showSecretsSync() {
```

Comment:
This check was used in several places but doing more than it needed to. For example, activation checks were relevant for top-level navigation like the side nav links, but not necessary for the `<Clients::Counts::NavBar>` component. Separating concerns seemed like the move here to make it easier to discern throughout the codebase what logic was actually necessary.

Reply:
I like separating the permissions vs has the feature. There are two things that I think make Secrets Sync a little different than the other nav items:
1. HVD should always show the sidebar nav
2. Secrets sync needs to be available for user that may not have permissions to secrets sync if the feature is not activated. Admins might not use Secrets Sync but they may need to activate it through the banner.

Reply:
Cool - so it sounds like i should remove the has nav permission check from the `cluster.hbs`